### PR TITLE
Add custom error pages for accountRecoveryEndpoint and authenticationEndpoint

### PR DESCRIPTION
**Pull request by GDRDABARERA**
Fixes https://github.com/wso2/product-is/issues/3021

### CODE REVIEW

File: components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint/src/main/webapp/WEB-INF/web.xml [hunk 1]
```diff
@@ -327,6 +327,55 @@
...
+        <error-code>503</error-code>
+        <location>/errors/error_503.html</location>
+    </error-page>
+    <error-page>
+        <error-code>504</error-code>
+        <location>/errors/error_504.html</location>
+    </error-page>
+    <error-page>
+        <location>/errors/error.html</location>
+    </error-page>
```
Comment (malithie) [inline components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint/src/main/webapp/WEB-INF/web.xml]: Check if this overrides the error page which is configured for exception.

File: components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint/src/main/webapp/WEB-INF/web.xml [hunk 2]
```diff
@@ -228,6 +228,55 @@
...
+        <error-code>503</error-code>
+        <location>/errors/error_503.html</location>
+    </error-page>
+    <error-page>
+        <error-code>504</error-code>
+        <location>/errors/error_504.html</location>
+    </error-page>
+    <error-page>
+        <location>/errors/error.html</location>
+    </error-page>
```
Comment (malithie) [inline components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint/src/main/webapp/WEB-INF/web.xml]: Check if this overrides the error page which is configured for exception.

File: components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint/src/main/webapp/WEB-INF/web.xml (hunk 1 above)
Comment (GDRDABARERA) [reply components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint/src/main/webapp/WEB-INF/web.xml]: No I checked throwing an exception, It gives the generic-exception-response.jsp, not the error.html

File: components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint/src/main/webapp/WEB-INF/web.xml (hunk 2 above)
Comment (GDRDABARERA) [reply components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint/src/main/webapp/WEB-INF/web.xml]: No it does not override. I checked throwing an exception, It gives the generic-exception-response.jsp, not the error.html

**Pull request merged**